Quotient/keyimport.h

```diff
@@ -35,10 +35,12 @@ class QUOTIENT_API KeyImport : public QObject
 
     Q_INVOKABLE Error importKeys(QString data, const QString& passphrase,
                                  const Quotient::Connection* connection);
+    Q_INVOKABLE Quotient::Expected<QByteArray, Error> exportKeys(const QString& passphrase, Quotient::Connection* connection);
```

Comment:
I think constifying won't interfere with `Q_INVOKABLE`, will it?
```suggestion
    Q_INVOKABLE Quotient::Expected<QByteArray, Error> exportKeys(const QString& passphrase, const Quotient::Connection* connection);
```